ts/Data/DataTableCore.ts

```diff
@@ -209,10 +212,42 @@ class DataTableCore {
         columnNames?: Array<string>
     ): (DataTable.Row|undefined) {
         return (columnNames || Object.keys(this.columns)).map(
-            (key): DataTableCore.CellType => this.columns[key]?.[rowIndex]
+            (key): DataTable.CellType => this.columns[key]?.[rowIndex]
         );
     }
 
+    /**
+     * Replaces a column reference in the table. It does not update the row
+     * count, but only replaces the reference to the column. Can also call the
+     * `afterSetColumns` event and update the table version tag if `silent` is
+     * set to `false`.
+     *
+     * @param {string} columnName
+     * Name of the column to replace.
+     *
+     * @param {Highcharts.DataTableColumn} column
+     * New column reference.
+     *
+     * @param {boolean} silent
+     * Whether to suppress events. Default is `true`.
+     *
+     * @emits #afterSetColumns
+     *
+     * @private
+     */
+    public replaceColumnRef(
```

Comment:
Can we just call it `replaceColumn`? Because that's what it does...

Reply:
I'll delete it since it's not needed anymore - all column array reference overrides are inside the `DataTableCore` class, so in this case encapsulation no longer forces the use of this method anywhere.